caas/kubernetes/provider/annotations/annotations.go

```diff
@@ -0,0 +1,120 @@
+// Copyright 2019 Canonical Ltd.
```

Comment:
this probably doesn't need it's own package

Reply:
This seems very heavy weight considering we only use it in a few places
It seems like a map would be ok and a lot less code
If this is needed, it really belongs in the core package

Reply:
moved to `core` package